### PR TITLE
Remove trailing whitespace from non-Scaladoc comments.

### LINKED ISSUE #149: Trailing whitespace is preserved in non-Scaladoc comments
Trailing whitespace is stripped from all source files, except within non-Scaladoc comments.

Scaladoc comments have trailing whitespace stripped.

PR coming with fix to remove trailing whitespace from all comments.

### PR DESCRIPTION
**Pull request by jkinkead**
Fixes #149 .

Note unit tests which will fail on `master`.

### CODE REVIEW

File: scalariform/src/test/scala/scalariform/formatter/CommentFormatterTest.scala [hunk 1]
```diff
@@ -99,6 +99,44 @@ class CommentFormatterTest extends AbstractFormatterTest {
     | */
     |"""
 
+  """/**
+    | * Trailing whitespace on this line and the line below should be stripped.   
+    | *
```
Comment (sschaef) [inline scalariform/src/test/scala/scalariform/formatter/CommentFormatterTest.scala]: We don't want to have trailing whitespace in our code. Please append a `$` sign at the end (or some other character) and strip it away afterwards with `str.replace("$", "")`

Comment (fommil) [inline scalariform/src/test/scala/scalariform/formatter/CommentFormatterTest.scala]: :+1: for 🐵 http://www.fileformat.info/info/unicode/char/1f435/index.htm

**Comment by jkinkead** [comment]
Replaced EOL space literals with unicode escapes.

**Comment by sschaef** [comment]
ok to test

**Comment by sschaef** [comment]
LGTM

**Pull request merged**